pkg/controller/migmigration/quiesce.go

```diff
@@ -88,25 +102,65 @@ func (t *Task) scaleDownDeploymentConfigs(client k8sclient.Client) error {
 }
 
 // Scales down all Deployments
-func (t *Task) scaleDownDeployments(client k8sclient.Client) error {
+func (t *Task) scaleDownDeployments(client k8sclient.Client, kubeVersion int) error {
+	if kubeVersion >= reference.AppsGap {
+		return t.scaleDownv1Deployments(client)
+	}
+	return t.scaleDownv1betaDeployments(client)
+}
+
+// Scales down all Deployments (kubernetes < 16)
+func (t *Task) scaleDownv1betaDeployments(client k8sclient.Client) error {
```

Comment:
I have the same question for here. If we are going with the approach Jeff outlined that automatically handles older versions, would these version-specialized methods still be necessary?